app/templates/components/tables/utilities/search-box.hbs

```diff
@@ -1,4 +1,4 @@
 <div class="ui {{if (eq @size "large") '' 'small'}} icon input search-box">
-  <Input @type="text" @value={{readonly this.searchQuery}} @keyUp={{action 'setSearchQuery' value="target.value"}} placeholder="{{t "Search"}} ..." />
+  <Input  class="{{ if this.device.isMobile "expand-bar" "" }}" @type="text" @value={{readonly this.searchQuery}} @keyUp={{action 'setSearchQuery' value="target.value"}} onfocus={{ if this.device.isMobile (action "showSessionFields")}} @focus-out={{if this.device.isMobile (action "showSessionFields")}}  placeholder="{{t "Search"}} ..." />
```

Comment:
we are using this component at some other places also. At mobile view we don't want changes. exmaple:
![new speaker](https://user-images.githubusercontent.com/72552281/103270543-2d821800-49de-11eb-8f8e-f69104c7c188.png)

You can use this condition _(and this.device.isMobile (eq this.session.currentRoute 'public.sessions'))_